src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -0,0 +1,112 @@
+package com.google.codeu.servlets;
+
+
+import java.io.IOException;
+
+
+import javax.servlet.annotation.WebServlet;
+
+import javax.servlet.http.HttpServlet;
+
+import javax.servlet.http.HttpServletRequest;
+
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.appengine.api.users.UserService;
+
+import com.google.appengine.api.users.UserServiceFactory;
+
+import com.google.codeu.data.Datastore;
+
+import com.google.codeu.data.User;
+
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+
+/**
+
+ * Responds with a hard-coded message for testing purposes.
+
+ */
+
+@WebServlet("/about")
+
+public class AboutMeServlet extends HttpServlet{
+
+  private Datastore datastore;
+
+  public void init(){
+
+    datastore = new Datastore();
+
+  }
+
+ @Override
+
+ public void doGet(HttpServletRequest request, HttpServletResponse response)
+ throws IOException
+ {
+
+   try
+{
+    // statement(s) that might cause exception
+
+
+  response.setContentType("text/html");
+  String user_email = request.getParameter("user");
+  if (user_email == null || user_email == ""){
```

Comment:
I'm not sure but I guess we don't need `user_email == null` condition as `request.getParameter("user");` will never return null, it'd return `""`. Kindly confirm. I'm not sure about that.

Reply:
Yes, I have seen certain posts where request.getParameter() returns null.